oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -58,12 +59,13 @@ def get_computer_system(uuid):
     """
 
     try:
-        resource = _get_oneview_resource(uuid)
+        resource_id, encl_id = zone.split_base_id_and_enclosure_id(uuid)
```

Comment:
I think we should rename "uuid" by "id" at  @computer_system.route("/redfish/v1/Systems/<id>", methods=["GET"])

Reply:
I will change to `system_id` to avoid shadowing of built-in name "`id`" of python.